bin/gltf-pipeline.js

```diff
@@ -49,7 +61,13 @@ var options = {
     binary : binary,
     embed : !separate,
     embedImage : !separateImage,
-    quantize : quantize
+    quantize : quantize,
+    ao_diffuse : ao_diffuse,
+    ao_scene : ao_scene,
+    ao_rayDepth : ao_rayDepth,
+    ao_resolution : ao_resolution,
+    ao_samples : ao_samples,
+    imageProcess : ao_diffuse
```

Comment:
Does `imageProcess` still need to be true if `--ao-separate` is set? Maybe this type of thing will be handled later.